base/process.jl

```diff
@@ -701,9 +701,29 @@ with a code of 0). An exception is raised if the process cannot be started.
 """
 success(cmd::AbstractCmd) = success(_spawn(cmd))
 
+struct ProcessExitedException
+    procs::Union{Vector{Process},Nothing}
+    # ProcessExitedException(nothing) is allowed for Distributed stdlib compat
+end
+ProcessExitedException(proc::Process) = ProcessExitedException([proc])
```

Comment:
You also need to define a constructor with no arguments in keeping with what was defined in Distributed.

Reply:
> I guess in theory someone might have been constructing `ProcessExitedError()`
in their own code.

Yes, this

Reply:
I'd prefer not to deprecate it anyway. `ProcessExitedException(nothing)` does not seem any more informative to me than `ProcessExitedException()`.

Reply:
Ok
Done